ui/app/models/pki/config.js

```diff
@@ -1,8 +1,9 @@
 import Model, { attr } from '@ember-data/model';
 import { inject as service } from '@ember/service';
 
-export default class PkiConfigImportModel extends Model {
+export default class PkiConfigModel extends Model {
   @service secretMountPath;
+  @attr('string') formType;
```

Comment:
since this determines the request url, it feels a little strange that it's an attribute on the config model.

Another option could be making it a `@tracked` property in the form component and then pass it to the save method via adapter options:

```js
// pki-configure-form.ts

@action submitForm() {
    const { formType } = this.args.config;
    if (!this.args.config) return;
    this.args.config
      .save({ adapterOptions: { formType } })
     ...
 }
```

but potato, potato! 🥔

Reply:
That's fair. I did this initially because I figured the dedicated `generate root` and `generate csr` routes would be able to set `formType` on the model and then the save actions would call the appropriate endpoints for free, but we can just as easily set the formType explicitly. 
I think this is related to something a different comment touched on, which is there are multiple endpoints to do any of these actions, and I'm a little nervous that we're going to get issues about how people have ACLs for one of the endpoints that we didn't use 😬

Reply:
Ahh, that makes sense. However you want to handle it! 

Ugh..yes. This would be a good thing to ask Alex about today